### DISCUSSION TITLE
I still can't subscribe to the following feed, and my list no longer has these displays

### DISCUSSION THREAD

**Discussion by ayuechuan** [community/community#66436] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

You want to resolve the dynamic subscription to your followers

**Comment by cwegener** [comment]
The following feed seemingly now needs to be accessed in a way that is not congruent with the documentation anymore.

Some kind soul sent me a direct link to the following feed, which I have been using for now.

https://github.com/dashboard-feed

**Comment by divyam751** [comment]


**Reply by esimov** [reply]
This is joke, right?

**Comment by icidasset** [comment]
I’m not seeing any activity from the people I follow either in my feed even though the stars filter is active (among all the other ones except recommendations).
